README.md

```diff
@@ -138,6 +138,12 @@ Initial design: https://uclahs.box.com/s/eejwmwmdky7wsfcrs8a3jijy70rh6atp
 ## license:
 Author: Gina Kim (ginakim@mednet.ucla.edu)
 
-Pipeval is currently free to use under the GPL 2 license, terms of which are under the "COPYING" file.
+PipeVal is licensed under the GNU General Public License version 2. See the file LICENSE for the terms of the GNU GPL license.
 
-Copyright (C) 2021 University of California Los Angeles ("Boutros Lab"). All rights reserved.
+PipeVal is a tool which can be used to validate the inputs and outputs of various bioinformatic pipelines.
+
+Copyright (C) 2022 University of California Los Angeles ("Boutros Lab") All rights reserved.
```

Comment:
Can you please check the date of the initial commit? I think this should be 2020-2022